dokan/dokan.c

```diff
@@ -249,6 +249,12 @@ int DOKANAPI DokanMain(PDOKAN_OPTIONS DokanOptions,
     return DOKAN_MOUNT_ERROR;
   }
 
+  
+  // wait until the device is realy mounted
+  if (DokanOptions->Options & DOKAN_OPTION_MOUNT_MANAGER) {
```

Comment:
The issue here is not related to Mount manager but to I/O manager. It is general to all Dokan mount method so this check shouldn't be done.